lib/faraday/adapter/my_adapter.rb

```diff
@@ -50,11 +50,11 @@ def call(env)
           end
         end
 
-      # NOTE: An adapter `call` MUST return the `env.response`. If `save_response` is the last line in your `call`
-      # method implementation, it will automatically return the response for you.
-      # Otherwise, you'll need to manually do so. You can do this with any (not both) of the following lines:
-      # * @app.call(env)
-      # * env.response
+        # NOTE: An adapter `call` MUST return the `env.response`. If `save_response` is the last line in your `call`
+        # method implementation, it will automatically return the response for you.
+        # Otherwise, you'll need to manually do so. You can do this with any (not both) of the following lines:
+        # * @app.call(env)
+        # * env.response
```

Comment:
These are confusing when followed by these other 2 comment lines 🤔
Can we leave an empty line in between, or move the comments related to the rescue inside the rescue maybe?